### PR TITLE
README.md - Prerequisites

### PR DESCRIPTION
**Pull request by dariodsa**
Prerequisites for using the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ This is an attempt to aggregate data about a house market and show them in some
...
+Fedora/CentOS/RedHat:   
+```
+# dnf install jq
+``` 
+Ubuntu:   
+```
+# apt-get install jq
+```
+
+Give examples
```
Comment (pschoffer) [inline README.md]: Give examples? Are the examples missing or did I misunderstood?

**Review by pschoffer — COMMENTED** [review]
Nice! I just had one question.

File: README.md (hunk 1 above)
Comment (dariodsa) [reply README.md]: I was a clipboard problem. It is ok now.

Comment (pschoffer) [reply README.md]: It is. Thanks!

**Pull request merged**